dangerzone/gui/main_window.py

```diff
@@ -509,7 +509,7 @@ def run(self):
             ocr_lang,
             self.stdout_callback,
         ):
-            self.finished.emit(self.error)
+            self.is_finished.emit(self.error)
```

Comment:
To fix the `finished` TypeError, we may want to instead use [`void QThread::exit`](https://doc.qt.io/qt-6.2/qthread.html#exit).

Reply:
```suggestion
```
If we remove this (as it doesn't seem to be needed), we avoid the typing issue altogether.

Reply:
Nice, looks good to me if that works on your end (I haven't tested it myself).

Reply:
Merged your change into this PR branch, see 5af3d73.